src/ec/packages/ec-component-ecl-compliance/components/ecl-compliance-description-list.html.twig

```diff
@@ -0,0 +1,44 @@
+{#
+  Parameters:
+  - "items" (array of objects)
+#}
+
+{% extends "@ecl-twig/ec-component-ecl-compliance/ecl-compliance.html.twig" %}
+
+{% set component = 'description-list' %}
+{% set message = '' %}
+{% set no_items_case = items is not defined or items is empty or items is not iterable %}
+{% set no_term_case = false %}
+{% set no_term_in_item_case = false %}
+{% set no_definition_case = false %}
+{% set no_definition_in_item_case = false %}
+
+{% if no_items_case == false %}
+  {% for item in items %}
+    {% if item.label is not defined or item.label is empty %}
```

Comment:
Copy-paste error most probably. No labels in this data structure. Not to mention `no_label_case` is not even used later.

Reply:
True, sorry actually for the status of this pr, i did not remember much about it but i surely remember that when working on this i noticed that the logic we have in the template is probably wrong, as it is also in ECL, i opened a ticket for this: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-1137
i probably got disilluded then about the possibility of define a "compliance" for this :)
Anyway for the sake of this implementation, i now covered the use cases i could identfy for it.

Reply:
Added reference to this comment in the mentioned ticket. I think we speak about different things, but left the reference for the future.
Thanks for removing the unnecessary part.